src/Command/App/NewCommand.php

```diff
@@ -15,15 +16,16 @@
  */
 class NewCommand extends CommandBase {
 
-  protected static $defaultName = 'new';
+  protected static $defaultName = 'app:new';
```

Comment:
This command doesn't create a new cloud app, which is what I'd expect a command named this to do.

Reply:
Aldo this will break the Cloud IDE Get Started page from which we have a CTA to create a new app.

Reply:
It shouldn't break anything since I added an alias for backwards compatibility.

So what do you want to do? Just undo the change (back to `acli new`), or rename it something like `acli app:new:local`? I would argue that `acli new` is even more confusing and ambiguous than `acli app:new`, we should add _some_ sort of context.

Reply:
I think `app:new:local` makes sense. Keep alias for `new`.